src/pages/vedtak/Vedtaksoppsummering.tsx

```diff
@@ -25,113 +17,40 @@ interface Props {
     sak: Sak;
 }
 
-const vedtaksresultatToTekst = (type: VedtakType, intl: IntlShape): string => {
-    switch (type) {
-        case VedtakType.SØKNAD:
-            return intl.formatMessage({ id: 'vedtaktype.søknad' });
-        case VedtakType.AVSLAG:
-            return intl.formatMessage({ id: 'vedtaktype.avslått' });
-        case VedtakType.ENDRING:
-            return intl.formatMessage({ id: 'vedtaktype.endring' });
-        case VedtakType.OPPHØR:
-            return intl.formatMessage({ id: 'vedtaktype.opphør' });
-        case VedtakType.INGEN_ENDRING:
-            return intl.formatMessage({ id: 'vedtaktype.ingenendring' });
-        case VedtakType.REGULER_GRUNNBELØP:
-            return intl.formatMessage({ id: 'vedtaktype.regulergrunnbeløp' });
-    }
-};
+function hentRevurderingOgKnyttetBehandling(sak: Sak, vedtak: Vedtak) {
+    const revurderingSomFørteTilVedtak = sak.revurderinger.find(
+        (b) => b.id === vedtak.behandlingId
+    ) as IverksattRevurdering;
+
+    const revurdertBehandling = sak.behandlinger.find(
```

Comment:
Hvordan vet vi at revurderingSomFørteTilVedtak ikke er en annen revurdering?

Reply:
Foreslår vi tar høyde for dette og heller returnerer GrunnlagsdataOgVilkårsvurderinger direkte

Reply:
Vet ikke om jag skjønte vad du menade men vi finner revurderingen som har den specificerte vedtaks-id:en.